core/templates/pages/collection-editor-page/modals/collection-editor-save-modal.component.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for collection editor save modal.
+ */
+
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { AppConstants } from 'app.constants';
+import { ConfirmOrCancelModal } from 'components/common-layout-directives/common-elements/confirm-or-cancel-modal.component';
+
+@Component({
+  selector: 'oppia-collection-editor-save-modal',
+  templateUrl: 'collection-editor-save-modal.component.html'
+})
+export class CollectionEditorSaveModalComponent extends ConfirmOrCancelModal {
+  isCollectionPrivate: boolean;
```

Comment:
Better to name this `collectionIsPrivate`.

Reply:
done